pyroclast/cpvae/model.py

```diff
@@ -0,0 +1,145 @@
+import numpy as np
+import sklearn.tree
+import tensorflow as tf
+import tensorflow_datasets as tfds
+import tensorflow_probability as tfp
+
+from pyroclast.common.tf_util import img_discretized_logistic_log_prob
+from pyroclast.cpvae.ddt import (get_decision_tree_boundaries,
+                                 transductive_box_inference)
+
+
+class CpVAE(tf.Module):
+
```

Comment:
Autodoc class description. Explicitly say what cpvae stands for once in the file.